#### Moderate correlation ( r > 0.3 ) & statistically significant ( p < 0.05 )
    receptor maps || genes
    
    serotonin: ELMO2, GADD45B, LANCL2, TRIM66, NEURL1B
    dopamine: STXBP6, GADD45B, LANCL2, RSPO3, NEURL1B
    gaba: GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B, STXBP6
    norepinephrine: NONE
    acetylcholine: GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B, STXBP6
    glutamate: GABB45B, LANCL2, RSPO3, TRIM66, NEURL1B
    mu opioid: STXBP6

In [25]:
import nibabel as nib
from neuromaps import nulls
from neuromaps import stats
from neuromaps import transforms
from neuromaps import datasets
from neuromaps.datasets import available_tags, available_annotations, fetch_annotation

In [107]:
import json
with open('data/data_desc.json', 'r') as f:
    data_desc = json.load(f)
data_desc.keys()

dict_keys(['dopamine', 'gaba', 'norepinephrine', 'acetylcholine', 'serotonin', 'glutamate', 'mu opioid'])

In [28]:
AD_association = nib.load('data/neurosynth_maps/terms/alzheimer_association-test_z_FDR_0.01.nii')

genes1 = {
    'ELMO2': nib.load('data/neurosynth_maps/genes1/gene_40559_ELMO2_63916.0.nii'),
    'GADD45B': nib.load('data/neurosynth_maps/genes1/gene_4590_GADD45B_4616.0.nii'),
    'LANCL2': nib.load('data/neurosynth_maps/genes1/gene_35202_LANCL2_55915.0.nii'),
    'RSPO3': nib.load('data/neurosynth_maps/genes1/gene_59060_RSPO3_84870.0.nii'),
    'TRIM66': nib.load('data/neurosynth_maps/genes1/gene_9722_TRIM66_9866.0.nii'),
    'DCAF5': nib.load('data/neurosynth_maps/genes1/gene_8739_DCAF5_8816.0.nii'),
    'ARL2': nib.load('data/neurosynth_maps/genes1/gene_399_ARL2_402.0.nii'),
    'NEURL1B': nib.load('data/neurosynth_maps/genes1/gene_33783_NEURL1B_54492.0.nii'),
    'STXBP6': nib.load('data/neurosynth_maps/genes1/gene_28827_STXBP6_29091.0.nii'),
    
}

In [30]:
def spatial_nulls(map1, map2):
    fsavg_map1 = transforms.mni152_to_fsaverage(map1, '10k')
    fsavg_map2 = transforms.mni152_to_fsaverage(map2, '10k')
    rotated = nulls.alexander_bloch(fsavg_map1, atlas='fsaverage', density='10k', n_perm=100, seed=1234)
    corr, pval = stats.compare_images(fsavg_map1, fsavg_map2, nulls=rotated)
    return (f'r = {corr:.3f}, p = {pval:.3f}')

### Serotonin

In [31]:
data_desc['serotonin']

{'gallezot2010': 'serotonin 5-HT(1B) receptor',
 'fazio2016': 'serotonin transporter',
 'beliveau2017': 'seratonin agonist',
 'radnakrishnan2018': '5-HT6 receptor availability'}

In [32]:
gallezot2010 = fetch_annotation(source='gallezot2010')
fazio2016 = fetch_annotation(source='fazio2016')
beliveau2017 = fetch_annotation(source='beliveau2017')
radnakrishnan2018 = fetch_annotation(source='radnakrishnan2018')

In [52]:
# correlation with all genes and gallezot2010

for gene in genes1:
    output = spatial_nulls(genes1[gene], gallezot2010)
    print(gene, '<-> gallezot2010: ', output)

ELMO2 <-> gallezot2010:  r = -0.301, p = 0.010
GADD45B <-> gallezot2010:  r = 0.026, p = 0.713
LANCL2 <-> gallezot2010:  r = -0.057, p = 0.396
RSPO3 <-> gallezot2010:  r = -0.095, p = 0.178
TRIM66 <-> gallezot2010:  r = -0.092, p = 0.366
DCAF5 <-> gallezot2010:  r = -0.065, p = 0.188
ARL2 <-> gallezot2010:  r = 0.121, p = 0.079
NEURL1B <-> gallezot2010:  r = 0.083, p = 0.257
STXBP6 <-> gallezot2010:  r = -0.008, p = 0.970


    Significant W/ gallezot2010: ELMO2

        ELMO2: LOW CORRELATION r = -0.301

In [53]:
# correlation with all genes and fazio2016

for gene in genes1:
    output = spatial_nulls(genes1[gene], fazio2016)
    print(gene, '<-> fazio2016: ', output)

ELMO2 <-> fazio2016:  r = -0.088, p = 0.129
GADD45B <-> fazio2016:  r = -0.072, p = 0.218
LANCL2 <-> fazio2016:  r = -0.010, p = 0.851
RSPO3 <-> fazio2016:  r = 0.047, p = 0.366
TRIM66 <-> fazio2016:  r = -0.084, p = 0.139
DCAF5 <-> fazio2016:  r = -0.003, p = 0.921
ARL2 <-> fazio2016:  r = -0.013, p = 0.881
NEURL1B <-> fazio2016:  r = 0.133, p = 0.020
STXBP6 <-> fazio2016:  r = 0.054, p = 0.277


    Significant W/ fazio2016: NEURL1B

        NEURL1B: VERY LOW CORRELATION r = 0.133

In [54]:
# multiple maps in beliveau2017

In [55]:
# correlation with all genes and beliveau2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], beliveau2017[('beliveau2017', 'az10419369', 'MNI152', '1mm')])
    print(gene, '<-> beliveau2017 1: ', output)

ELMO2 <-> beliveau2017 1:  r = -0.126, p = 0.010
GADD45B <-> beliveau2017 1:  r = 0.070, p = 0.069
LANCL2 <-> beliveau2017 1:  r = -0.102, p = 0.020
RSPO3 <-> beliveau2017 1:  r = -0.059, p = 0.119
TRIM66 <-> beliveau2017 1:  r = 0.013, p = 0.782
DCAF5 <-> beliveau2017 1:  r = -0.059, p = 0.030
ARL2 <-> beliveau2017 1:  r = -0.002, p = 0.901
NEURL1B <-> beliveau2017 1:  r = -0.012, p = 0.683
STXBP6 <-> beliveau2017 1:  r = -0.016, p = 0.703


    Significant W/ beliveau2017 #1: ELMO2, LANCL2, DCAF5

        ELMO2: VERY LOW CORRELATION r = -0.126
        LANCL2: VERY LOW CORRELATION r = -0.102
        DCAF5: VERY LOW CORRELATION r = -0.059

In [56]:
# correlation with all genes and beliveau2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], beliveau2017[('beliveau2017', 'cimbi36', 'MNI152', '1mm')])
    print(gene, '<-> beliveau2017 2: ', output)

ELMO2 <-> beliveau2017 2:  r = -0.065, p = 0.059
GADD45B <-> beliveau2017 2:  r = -0.065, p = 0.069
LANCL2 <-> beliveau2017 2:  r = 0.012, p = 0.762
RSPO3 <-> beliveau2017 2:  r = 0.043, p = 0.257
TRIM66 <-> beliveau2017 2:  r = -0.012, p = 0.762
DCAF5 <-> beliveau2017 2:  r = -0.030, p = 0.208
ARL2 <-> beliveau2017 2:  r = -0.025, p = 0.574
NEURL1B <-> beliveau2017 2:  r = 0.072, p = 0.050
STXBP6 <-> beliveau2017 2:  r = -0.020, p = 0.653


    Significant W/ beliveau2017 #2: NONE

In [57]:
# correlation with all genes and beliveau2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], beliveau2017[('beliveau2017', 'cumi101', 'MNI152', '1mm')])
    print(gene, '<-> beliveau2017 3: ', output)

ELMO2 <-> beliveau2017 3:  r = 0.110, p = 0.069
GADD45B <-> beliveau2017 3:  r = -0.234, p = 0.010
LANCL2 <-> beliveau2017 3:  r = 0.170, p = 0.010
RSPO3 <-> beliveau2017 3:  r = 0.261, p = 0.010
TRIM66 <-> beliveau2017 3:  r = -0.125, p = 0.020
DCAF5 <-> beliveau2017 3:  r = 0.054, p = 0.089
ARL2 <-> beliveau2017 3:  r = -0.054, p = 0.634
NEURL1B <-> beliveau2017 3:  r = 0.195, p = 0.010
STXBP6 <-> beliveau2017 3:  r = 0.010, p = 0.842


    Significant W/ beliveau2017 #3: GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B

        GADD45B: LOW CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        RSPO3: LOW CORRELATION
        TRIM66: LOW CORRELATION
        NEURL1B: LOW CORRELATION

In [58]:
# correlation with all genes and beliveau2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], beliveau2017[('beliveau2017', 'dasb', 'MNI152', '1mm')])
    print(gene, '<-> beliveau2017 4: ', output)

ELMO2 <-> beliveau2017 4:  r = 0.027, p = 0.673
GADD45B <-> beliveau2017 4:  r = -0.026, p = 0.752
LANCL2 <-> beliveau2017 4:  r = -0.014, p = 0.782
RSPO3 <-> beliveau2017 4:  r = 0.108, p = 0.040
TRIM66 <-> beliveau2017 4:  r = -0.084, p = 0.218
DCAF5 <-> beliveau2017 4:  r = 0.021, p = 0.574
ARL2 <-> beliveau2017 4:  r = -0.048, p = 0.634
NEURL1B <-> beliveau2017 4:  r = 0.079, p = 0.158
STXBP6 <-> beliveau2017 4:  r = 0.021, p = 0.644


    Significant W/ beliveau2017 #4: RSPO3

        RSPO3: LOW CORRELATION

In [59]:
# correlation with all genes and beliveau2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], beliveau2017[('beliveau2017', 'sb207145', 'MNI152', '1mm')])
    print(gene, '<-> beliveau2017 5: ', output)

ELMO2 <-> beliveau2017 5:  r = 0.010, p = 0.842
GADD45B <-> beliveau2017 5:  r = -0.136, p = 0.010
LANCL2 <-> beliveau2017 5:  r = 0.091, p = 0.020
RSPO3 <-> beliveau2017 5:  r = 0.130, p = 0.010
TRIM66 <-> beliveau2017 5:  r = -0.033, p = 0.465
DCAF5 <-> beliveau2017 5:  r = 0.012, p = 0.604
ARL2 <-> beliveau2017 5:  r = -0.043, p = 0.535
NEURL1B <-> beliveau2017 5:  r = 0.110, p = 0.010
STXBP6 <-> beliveau2017 5:  r = 0.005, p = 0.960


    Significant W/ beliveau2017 #5: GADD45B, LANCL2, RSPO3, NEURL1B

        GADD45B: LOW CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        NEURL1B: LOW CORRELATION

In [60]:
# correlation with all genes and radnakrishnan2018

for gene in genes1:
    output = spatial_nulls(genes1[gene], radnakrishnan2018)
    print(gene, '<-> radnakrishnan2018: ', output)

ELMO2 <-> radnakrishnan2018:  r = -0.221, p = 0.010
GADD45B <-> radnakrishnan2018:  r = -0.335, p = 0.010
LANCL2 <-> radnakrishnan2018:  r = 0.338, p = 0.010
RSPO3 <-> radnakrishnan2018:  r = 0.218, p = 0.020
TRIM66 <-> radnakrishnan2018:  r = -0.315, p = 0.010
DCAF5 <-> radnakrishnan2018:  r = 0.052, p = 0.376
ARL2 <-> radnakrishnan2018:  r = 0.190, p = 0.010
NEURL1B <-> radnakrishnan2018:  r = 0.375, p = 0.010
STXBP6 <-> radnakrishnan2018:  r = -0.115, p = 0.267


    Significant W/ radnakrishnan2018: ELMO2, GADD45B, LANCL2, RSPO3, TRIM66, ARL2, NEURL1B

        ELMO2: LOW CORRELATION
        GADD45B: LOW CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        TRIM66: LOW CORRELATION
        ARL2: LOW CORRELATION
        NEURL1B: LOW/MOD CORRELATION

### Dopamine

In [61]:
data_desc['dopamine']

{'sandiego2015': 'D2 dopamine receptors',
 'sasaki2012': 'dopamine transporter',
 'kaller2017': 'dopamine D1 receptors',
 'alarkurtti2015': 'raclopride - dopamine receptor'}

In [62]:
sandiego2015 = fetch_annotation(source='sandiego2015')
sasaki2012 = fetch_annotation(source='sasaki2012')
kaller2017 = fetch_annotation(source='kaller2017')
alarkurtti2015 = fetch_annotation(source='alarkurtti2015')

In [63]:
# correlation with all genes and sandiego2015

for gene in genes1:
    output = spatial_nulls(genes1[gene], sandiego2015)
    print(gene, '<-> sandiego2015: ', output)

ELMO2 <-> sandiego2015:  r = 0.140, p = 0.069
GADD45B <-> sandiego2015:  r = 0.111, p = 0.129
LANCL2 <-> sandiego2015:  r = -0.156, p = 0.010
RSPO3 <-> sandiego2015:  r = -0.109, p = 0.178
TRIM66 <-> sandiego2015:  r = 0.242, p = 0.010
DCAF5 <-> sandiego2015:  r = 0.015, p = 0.772
ARL2 <-> sandiego2015:  r = -0.123, p = 0.267
NEURL1B <-> sandiego2015:  r = -0.141, p = 0.040
STXBP6 <-> sandiego2015:  r = 0.302, p = 0.010


    Significant W/ sandiego2015: LANCL2, TRIM66, NEURL1B, STXBP6

        LANCL2: LOW CORRELATION
        TRIM66: LOW CORRELATION
        NEURL1B: LOW CORRELATION
        STXBP6: LOW CORRELATION

In [64]:
# correlation with all genes and sasaki2012

for gene in genes1:
    output = spatial_nulls(genes1[gene], sasaki2012)
    print(gene, '<-> sasaki2012: ', output)

ELMO2 <-> sasaki2012:  r = -0.037, p = 0.406
GADD45B <-> sasaki2012:  r = 0.026, p = 0.574
LANCL2 <-> sasaki2012:  r = 0.004, p = 0.931
RSPO3 <-> sasaki2012:  r = -0.076, p = 0.188
TRIM66 <-> sasaki2012:  r = 0.000, p = 0.990
DCAF5 <-> sasaki2012:  r = 0.047, p = 0.228
ARL2 <-> sasaki2012:  r = 0.061, p = 0.208
NEURL1B <-> sasaki2012:  r = -0.001, p = 0.970
STXBP6 <-> sasaki2012:  r = 0.177, p = 0.010


    Significant W/ sasaki2012: STXBP6

        STXBP6: LOW CORRELATION

In [65]:
# correlation with all genes and kaller2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], kaller2017)
    print(gene, '<-> kaller2017: ', output)

ELMO2 <-> kaller2017:  r = -0.045, p = 0.455
GADD45B <-> kaller2017:  r = -0.403, p = 0.010
LANCL2 <-> kaller2017:  r = 0.314, p = 0.010
RSPO3 <-> kaller2017:  r = 0.366, p = 0.010
TRIM66 <-> kaller2017:  r = -0.293, p = 0.010
DCAF5 <-> kaller2017:  r = 0.112, p = 0.040
ARL2 <-> kaller2017:  r = -0.020, p = 0.733
NEURL1B <-> kaller2017:  r = 0.431, p = 0.010
STXBP6 <-> kaller2017:  r = -0.177, p = 0.040


    Significant W/ kaller2017: GADD45B, LANCL2, RSPO3, TRIM66, DCAF5, NEURL1B, STXBP6

        GADD45B: MODERATE CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        TRIM66: LOW CORRELATION
        DCAF5: LOW CORRELATION
        NEURL1B: MODERATE CORRELATION
        STXBP6: LOW CORRELATION

In [66]:
# correlation with all genes and alarkurtti2015

for gene in genes1:
    output = spatial_nulls(genes1[gene], alarkurtti2015)
    print(gene, '<-> alarkurtti2015: ', output)

ELMO2 <-> alarkurtti2015:  r = -0.081, p = 0.208
GADD45B <-> alarkurtti2015:  r = 0.027, p = 0.743
LANCL2 <-> alarkurtti2015:  r = 0.027, p = 0.673
RSPO3 <-> alarkurtti2015:  r = -0.100, p = 0.149
TRIM66 <-> alarkurtti2015:  r = 0.012, p = 0.782
DCAF5 <-> alarkurtti2015:  r = 0.072, p = 0.040
ARL2 <-> alarkurtti2015:  r = 0.071, p = 0.396
NEURL1B <-> alarkurtti2015:  r = -0.017, p = 0.812
STXBP6 <-> alarkurtti2015:  r = 0.038, p = 0.594


    Significant W/ alarkurtti2015: DCAF5

        DCAF5: LOW CORRELATION

### Gaba

In [67]:
data_desc['gaba']

{'norgaard2020': 'GABAA receptors', 'dukart2018': 'flumazenil gaba anatgonist'}

In [68]:
norgaard2020 = fetch_annotation(source='norgaard2020')
dukart2018 = fetch_annotation(source='dukart2018')

In [69]:
# correlation with all genes and norgaard2020

for gene in genes1:
    output = spatial_nulls(genes1[gene], norgaard2020)
    print(gene, '<-> norgaard2020: ', output)

ELMO2 <-> norgaard2020:  r = -0.168, p = 0.010
GADD45B <-> norgaard2020:  r = -0.121, p = 0.040
LANCL2 <-> norgaard2020:  r = 0.125, p = 0.040
RSPO3 <-> norgaard2020:  r = 0.068, p = 0.257
TRIM66 <-> norgaard2020:  r = -0.207, p = 0.010
DCAF5 <-> norgaard2020:  r = 0.008, p = 0.921
ARL2 <-> norgaard2020:  r = 0.102, p = 0.030
NEURL1B <-> norgaard2020:  r = 0.169, p = 0.010
STXBP6 <-> norgaard2020:  r = -0.182, p = 0.059


    Significant W/ norgaard2020: ELMO2, GADD45B, LANCL2, TRIM66, ARL2, NEURL1B

        ELMO2: LOW CORRELATION
        GADD45B: LOW CORRELATION
        LANCL2: LOW CORRELATION
        TRIM66: LOW CORRELATION
        ARL2: LOW CORRELATION
        NEURL1B: LOW CORRELATION

In [71]:
# correlation with all genes and dukart2018 

for gene in genes1:
    output = spatial_nulls(genes1[gene], dukart2018[('dukart2018', 'flumazenil', 'MNI152', '3mm')])
    print(gene, '<-> dukart2018 1: ', output)

ELMO2 <-> dukart2018 1:  r = -0.185, p = 0.030
GADD45B <-> dukart2018 1:  r = -0.377, p = 0.010
LANCL2 <-> dukart2018 1:  r = 0.337, p = 0.010
RSPO3 <-> dukart2018 1:  r = 0.312, p = 0.010
TRIM66 <-> dukart2018 1:  r = -0.394, p = 0.010
DCAF5 <-> dukart2018 1:  r = 0.051, p = 0.436
ARL2 <-> dukart2018 1:  r = 0.108, p = 0.119
NEURL1B <-> dukart2018 1:  r = 0.423, p = 0.010
STXBP6 <-> dukart2018 1:  r = -0.304, p = 0.010


    Significant W/ dukart2018 #1: ELMO2, GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B, STXBP6

        ELMO2: LOW CORRELATION
        GADD45B: MODERATE CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        TRIM66: MODERATE CORRELATION
        NEURL1B: MODERATE CORRELATION
        STXBP6: LOW CORRELATION

In [73]:
# correlation with all genes and dukart2018

for gene in genes1:
    output = spatial_nulls(genes1[gene], dukart2018[('dukart2018', 'fpcit', 'MNI152', '3mm')])
    print(gene, '<-> dukart2018 2: ', output)

ELMO2 <-> dukart2018 2:  r = 0.155, p = 0.089
GADD45B <-> dukart2018 2:  r = 0.247, p = 0.030
LANCL2 <-> dukart2018 2:  r = -0.318, p = 0.010
RSPO3 <-> dukart2018 2:  r = -0.187, p = 0.040
TRIM66 <-> dukart2018 2:  r = 0.330, p = 0.010
DCAF5 <-> dukart2018 2:  r = -0.012, p = 0.881
ARL2 <-> dukart2018 2:  r = -0.237, p = 0.010
NEURL1B <-> dukart2018 2:  r = -0.239, p = 0.010
STXBP6 <-> dukart2018 2:  r = 0.258, p = 0.030


    Significant W/ dukart2018 #2: GADD45B, LANCL2, RSPO3, TRIM66, ARL2, NEURL1B, STXBP6

        GADD45B: LOW CORRELATION
        LANCL2: LOW CORRELATION
        RSPO3: LOW CORRELATION
        TRIM66: LOW CORRELATION
        ARL2: LOW CORRELATION
        NEURL1B: LOW CORRELATION
        STXBP6: LOW CORRELATION

### Norepinephrine

In [77]:
data_desc['norepinephrine']

{'hesse2017': 'noradrenaline transporter'}

In [78]:
hesse2017 = fetch_annotation(source='hesse2017')

In [76]:
# correlation with all genes and hesse2017

for gene in genes1:
    output = spatial_nulls(genes1[gene], hesse2017)
    print(gene, '<-> hesse2017: ', output)

ELMO2 <-> hesse2017:  r = -0.113, p = 0.079
GADD45B <-> hesse2017:  r = -0.168, p = 0.050
LANCL2 <-> hesse2017:  r = 0.222, p = 0.010
RSPO3 <-> hesse2017:  r = 0.111, p = 0.119
TRIM66 <-> hesse2017:  r = -0.240, p = 0.010
DCAF5 <-> hesse2017:  r = 0.020, p = 0.653
ARL2 <-> hesse2017:  r = 0.184, p = 0.010
NEURL1B <-> hesse2017:  r = 0.158, p = 0.020
STXBP6 <-> hesse2017:  r = -0.116, p = 0.208


    Significant W/ hesse2017: GADD45B, LANCL2, TRIM66, ARL2, NEURL1B

### Acetylcholine

In [80]:
data_desc['acetylcholine']

{'tuominen': 'acetylcholine transporter density',
 'hillmer2016': 'nicotinic acetylcholine receptors',
 'naganawa2020': 'M1 muscarinic acetylcholine receptors'}

In [81]:
tuominen = fetch_annotation(source='tuominen')
hillmer2016 = fetch_annotation(source='hillmer2016')
naganawa2020 = fetch_annotation(source='naganawa2020')

In [82]:
# correlation with all genes and tuominen

for gene in genes1:
    output = spatial_nulls(genes1[gene], tuominen)
    print(gene, '<-> tuominen: ', output)

ELMO2 <-> tuominen:  r = 0.071, p = 0.406
GADD45B <-> tuominen:  r = 0.245, p = 0.010
LANCL2 <-> tuominen:  r = -0.262, p = 0.010
RSPO3 <-> tuominen:  r = -0.225, p = 0.020
TRIM66 <-> tuominen:  r = 0.232, p = 0.010
DCAF5 <-> tuominen:  r = -0.020, p = 0.683
ARL2 <-> tuominen:  r = -0.088, p = 0.248
NEURL1B <-> tuominen:  r = -0.222, p = 0.010
STXBP6 <-> tuominen:  r = 0.294, p = 0.010


    Significant W/ tuominen: GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B, STXBP6

In [83]:
# correlation with all genes and hillmer2016

for gene in genes1:
    output = spatial_nulls(genes1[gene], hillmer2016)
    print(gene, '<-> hillmer2016: ', output)

ELMO2 <-> hillmer2016:  r = 0.015, p = 0.792
GADD45B <-> hillmer2016:  r = 0.336, p = 0.010
LANCL2 <-> hillmer2016:  r = -0.333, p = 0.010
RSPO3 <-> hillmer2016:  r = -0.344, p = 0.010
TRIM66 <-> hillmer2016:  r = 0.371, p = 0.010
DCAF5 <-> hillmer2016:  r = -0.078, p = 0.129
ARL2 <-> hillmer2016:  r = -0.051, p = 0.347
NEURL1B <-> hillmer2016:  r = -0.356, p = 0.010
STXBP6 <-> hillmer2016:  r = 0.340, p = 0.010


    Significant W/ hillmer2016: GADD45B, LANCL2, RSPO3, TRIM66, NEURL1B, STXBP6

In [84]:
# correlation with all genes and naganawa2020
for gene in genes1:
    output = spatial_nulls(genes1[gene], naganawa2020)
    print(gene, '<-> naganawa2020: ', output)

ELMO2 <-> naganawa2020:  r = -0.219, p = 0.010
GADD45B <-> naganawa2020:  r = -0.327, p = 0.010
LANCL2 <-> naganawa2020:  r = 0.338, p = 0.010
RSPO3 <-> naganawa2020:  r = 0.235, p = 0.020
TRIM66 <-> naganawa2020:  r = -0.324, p = 0.010
DCAF5 <-> naganawa2020:  r = 0.059, p = 0.297
ARL2 <-> naganawa2020:  r = 0.183, p = 0.010
NEURL1B <-> naganawa2020:  r = 0.374, p = 0.010
STXBP6 <-> naganawa2020:  r = -0.141, p = 0.139


    Significant W/ naganawa2020: ELMO2, GADD45B, LANCL2, RSPO3, TRIM66, ARL2, NEURL1B

### Glutamate

In [85]:
data_desc['glutamate']

{'rosaneto': 'mGluR5 ligand', 'smart2019': 'mGluR5'}

In [86]:
rosaneto = fetch_annotation(source='rosaneto')
smart2019 = fetch_annotation(source='smart2019')

In [87]:
# correlation with all genes and rosaneto

for gene in genes1:
    output = spatial_nulls(genes1[gene], rosaneto)
    print(gene, '<-> rosaneto: ', output)

ELMO2 <-> rosaneto:  r = -0.158, p = 0.050
GADD45B <-> rosaneto:  r = -0.421, p = 0.010
LANCL2 <-> rosaneto:  r = 0.396, p = 0.010
RSPO3 <-> rosaneto:  r = 0.334, p = 0.010
TRIM66 <-> rosaneto:  r = -0.391, p = 0.010
DCAF5 <-> rosaneto:  r = 0.069, p = 0.287
ARL2 <-> rosaneto:  r = 0.145, p = 0.030
NEURL1B <-> rosaneto:  r = 0.441, p = 0.010
STXBP6 <-> rosaneto:  r = -0.225, p = 0.050


    Significant W/ rosaneto: ELMO2, GADD45B, LANCL2, RSPO3, TRIM66, ARL2, NEURL1B, STXBP6

In [88]:
# correlation with all genes and smart2019

for gene in genes1:
    output = spatial_nulls(genes1[gene], smart2019)
    print(gene, '<-> smart2019: ', output)

ELMO2 <-> smart2019:  r = -0.138, p = 0.059
GADD45B <-> smart2019:  r = -0.393, p = 0.010
LANCL2 <-> smart2019:  r = 0.387, p = 0.010
RSPO3 <-> smart2019:  r = 0.336, p = 0.010
TRIM66 <-> smart2019:  r = -0.338, p = 0.010
DCAF5 <-> smart2019:  r = 0.089, p = 0.109
ARL2 <-> smart2019:  r = 0.139, p = 0.030
NEURL1B <-> smart2019:  r = 0.415, p = 0.010
STXBP6 <-> smart2019:  r = -0.162, p = 0.079


    Significant W/ smart2019: GADD45B, LANCL2, RSPO3, TRIM66, ARL2, NEURL1B

### Mu opioid

In [89]:
data_desc['mu opioid']

{'turtonen2020': 'carfentanil: mu opioid receptor availability',
 'kantonen2020': 'carfentanil: mu opioid receptor availability'}

In [90]:
turtonen2020 = fetch_annotation(source='turtonen2020')
kantonen2020 = fetch_annotation(source='kantonen2020')

In [91]:
# correlation with all genes and turtonen2020

for gene in genes1:
    output = spatial_nulls(genes1[gene], turtonen2020)
    print(gene, '<-> turtonen2020: ', output)

ELMO2 <-> turtonen2020:  r = 0.154, p = 0.050
GADD45B <-> turtonen2020:  r = -0.073, p = 0.347
LANCL2 <-> turtonen2020:  r = 0.019, p = 0.832
RSPO3 <-> turtonen2020:  r = 0.071, p = 0.356
TRIM66 <-> turtonen2020:  r = 0.141, p = 0.020
DCAF5 <-> turtonen2020:  r = 0.058, p = 0.287
ARL2 <-> turtonen2020:  r = -0.083, p = 0.515
NEURL1B <-> turtonen2020:  r = 0.026, p = 0.782
STXBP6 <-> turtonen2020:  r = 0.326, p = 0.010


    Significant W/ turtonen2020: ELMO2, TRIM66, STXBP6

In [92]:
# correlation with all genes and kantonen2020

for gene in genes1:
    output = spatial_nulls(genes1[gene], kantonen2020)
    print(gene, '<-> kantonen2020: ', output)

ELMO2 <-> kantonen2020:  r = 0.128, p = 0.089
GADD45B <-> kantonen2020:  r = -0.238, p = 0.020
LANCL2 <-> kantonen2020:  r = 0.172, p = 0.020
RSPO3 <-> kantonen2020:  r = 0.224, p = 0.010
TRIM66 <-> kantonen2020:  r = -0.004, p = 0.970
DCAF5 <-> kantonen2020:  r = 0.101, p = 0.050
ARL2 <-> kantonen2020:  r = -0.033, p = 0.812
NEURL1B <-> kantonen2020:  r = 0.191, p = 0.020
STXBP6 <-> kantonen2020:  r = 0.255, p = 0.020


    Significant W/ kantonen2020: GADD45B, LANCL2, RSPO3, DCAF5, NEURL1B, STXBP6